packages/manager/modules/pci/src/projects/project/edit/remove/remove.controller.js

```diff
@@ -0,0 +1,41 @@
+import { MESSAGES_CONTAINER_NAME } from '../edit.constant';
+
+export default class {
+  /* @ngInject */
+  constructor($state, $stateParams, $translate, CucCloudMessage, OvhApiCloudProject) {
+    this.$state = $state;
+    this.$translate = $translate;
+    this.CucCloudMessage = CucCloudMessage;
+    this.OvhApiCloudProject = OvhApiCloudProject;
+    this.isLoading = false;
+    this.serviceName = $stateParams.projectId;
+  }
+
+  remove() {
+    this.isLoading = true;
```

Comment:
Shouldn't be `isDeleting`?

```suggestion
    this.isDeleting = true;
```